### PR TITLE
style: Adding intellij darcula theme

### PR DESCRIPTION
**Pull request by manuelzzz**
### My base theme was:
image

## After implements in .css file on project:
image
image
image
image

### i hope you all like :)

### CODE REVIEW

File: frontend/static/themes/_list.json [hunk 1]
```diff
@@ -1181,6 +1181,13 @@
     "mainColor": "#363636",
     "subColor": "#4f4f4f",
     "textColor": "#1f1f1f"
+  },
+  {
+    "name": "intellij darcula",
```
Comment (Miodec) [inline frontend/static/themes/_list.json:1186]: Names must be the same as file names. Replace the space with an underscore

**Review by Miodec — CHANGES_REQUESTED** [review]
image

Not the best color choice so far, sub alt color cannot be this light - it should be much closer to the bg color. Also the contrast between text and sub color is too low. I would also swap text and sub around.

Please use other themes as your guide as to what which colors play what role.

**Comment by manuelzzz** [comment]
I think that is more acceptable
image

**Comment by Miodec** [comment]
Text color cannot be colored. It needs to be close to white or black (can have a slight tint).

**Comment by manuelzzz** [comment]
If i invert the sub and text colors make sense? becouse that exactly the color pallet

**Comment by Miodec** [comment]
It may be better. But i stil havent checked for similar themes that are already available.

**Comment by Miodec** [comment]
I think the sub and main colors are too simmilar to each other.

**Comment by manuelzzz** [comment]
Okay, i will change to better

**Pull request closed without merging**